collections/_drafts/agent-forwarding.md

```diff
@@ -1,13 +1,12 @@
 ---
 title: OpenINF Next-Gen Guidance on Agent Forwarding
 category: contributing
-authors: @OpenINF/wg-a-team
 published: false
 permalink:
```

Comment:
```suggestion
permalink: docs/internal/agent-forwarding
```

one moment thinking about this

Reply:
🔗 https://github.com/OpenINF/openinf.github.io/blob/46ac73115ca2c19dcaf0a6ab02ea66675af8e0d7/build/tasks/compile/siteify-health-files.mjs#L97

Reply:
something like that for now (we can change it later)